src/Microsoft.OpenApi.Readers/Document.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. 
+
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.OpenApi.Readers
```

Comment:
Move this to the Microsoft.OpenAPI namespace so that a user doesn't have to pull in a second namespace just to be able to load a document.